components/sup/src/main.rs

```diff
@@ -288,6 +292,8 @@ async fn split_apart_sup_run(sup_run: SupRun,
                               update_url: bldr_url.clone(),
                               update_channel: shared_load.channel.clone(),
                               http_disable: sup_run.http_disable,
+                              http_startup_timeout:
+                                  HttpStartupTimeout::configured_value().into(),
```

Comment:
If you keep `http_startup_timeout` typed as `HttpStartupTimeout`, this could just be `HttpStartupTimeout::configured_value()`; you only really need the `into()` at the point where it's actually time to wait. You don't lose anything, and you actually make it easier to track all the information flows and make sure they're consistent.

Reply:
Nice! Applied in bddeedd7fa6522fa2d2b1592d9518c80e19b6cf6 @christophermaier